test/exchange_adapters/bitmart.test.ts

```diff
@@ -0,0 +1,190 @@
+import { BitMartAdapter } from '../../src/exchange_adapters/bitmart'
+import { ExchangeAdapterConfig } from '../../src/exchange_adapters/base'
+import { baseLogger } from '../../src/default_config'
+import { Exchange, ExternalCurrency } from '../../src/utils'
+import BigNumber from 'bignumber.js'
+
+describe('BitMart adapter', () => {
+  let bitmartAdapter: BitMartAdapter
+
+  const config: ExchangeAdapterConfig = {
+    baseCurrency: ExternalCurrency.EUROC,
+    baseLogger,
+    quoteCurrency: ExternalCurrency.USDC,
+  }
+
+  beforeEach(() => {
+    bitmartAdapter = new BitMartAdapter(config)
+  })
+
+  afterEach(() => {
+    jest.clearAllTimers()
+    jest.clearAllMocks()
+  })
+
+  const validMockTickerJson = {
+    message: 'OK',
+    code: 1000,
+    trace: '48cff315816f4e1aa26ca72cccb46051.58.16892401718550155',
+    data: {
+      symbol: 'EUROC_USDC',
+      last_price: '1.10806017',
+      quote_volume_24h: '91361.42293780',
+      base_volume_24h: '82758.2',
+      high_24h: '1.10872025',
+      low_24h: '1.09676936',
+      open_24h: '1.09735661',
+      close_24h: '1.10806017',
+      best_ask: '1.11109875',
+      best_ask_size: '51.1',
+      best_bid: '1.10756051',
+      best_bid_size: '55.2',
+      fluctuation: '+0.0098',
+      timestamp: 1689239976811,
+      url: 'https://www.bitmart.com/trade?symbol=EUROC_USDC',
+    },
+  }
+
+  const inValidMockTickerJson = {
+    message: 'OK',
+    code: 1000,
+    trace: '48cff315816f4e1aa26ca72cccb46051.58.16892401718550155',
+    data: {
+      symbol: 'EUROC_USDC',
+      last_price: undefined,
+      quote_volume_24h: '91361.42293780',
+      base_volume_24h: undefined,
+      high_24h: '1.10872025',
+      low_24h: '1.09676936',
+      open_24h: '1.09735661',
+      close_24h: '1.10806017',
+      best_ask: undefined,
+      best_ask_size: '51.1',
+      best_bid: undefined,
+      best_bid_size: '55.2',
+      fluctuation: '+0.0098',
+      timestamp: undefined,
+      url: 'https://www.bitmart.com/trade?symbol=EUROC_USDC',
+    },
+  }
+
+  describe('parseTicker', () => {
+    it('handles a response that matches the documentation', () => {
+      const ticker = bitmartAdapter.parseTicker(validMockTickerJson)
+      expect(ticker).toEqual({
+        source: Exchange.BITMART,
+        symbol: bitmartAdapter.standardPairSymbol,
+        ask: new BigNumber(1.11109875),
+        baseVolume: new BigNumber(82758.2),
+        bid: new BigNumber(1.10756051),
+        lastPrice: new BigNumber(1.10806017),
```

Comment:
can you change this value in `validMockTickerJson`? Is currently the same as `close_24h`, which is not great for testing purposes